research-software-directory/RSD-as-a-service
# Admin role in database

## Save admins in database

Changes proposed in this pull request:

* RSD admins are now defined in a new table called `admin_account`.
* Consequently, the env variable `RSD_ADMIN_EMAIL_LIST` is dropped.


How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=0`
* Log in in your preferred way. You should not have access to any admin features. Go to your settings to grab your account ID.
* Connect to the database and execute the following query: `INSERT INTO admin_account VALUES ('00000000-0000-0000-0000-000000000000');`, where you replace the UUID with the value you got in the previous step.
* Log out and log in with the same account. You should now have access to all admin features.
* Log in with a different account. You should not have admin access.
* Repeatedly logging in and out with different accounts should work.


PR Checklist:
